x/claim/keeper/msg_server_claim_eth.go

```diff
@@ -163,3 +171,26 @@ func hexDecode(s string) ([]byte, error) {
 func has0xPrefix(str string) bool {
 	return strings.HasPrefix(str, "0x") || strings.HasPrefix(str, "0X")
 }
+
+func mergeClaimRecords(claimA types.ClaimRecord, claimB types.ClaimRecord) (types.ClaimRecord, error) {
+	if claimA == (types.ClaimRecord{}) {
```

Comment:
Can we please add a method like this to check , we should ensure that the amount is not negative , and also it is not zero , and the coin denom is legit

```
func(m *ClaimRecord) IsEmpty() bool{
	if m == nil {
		return true
	}
	if m.Address == "" {
		return true
	}

	if !m.AmountClaim.IsNil() && m.AmountClaim.IsValid() && !m.AmountClaim.IsZero() {
		return false
	}
	if !m.AmountVote.IsNil() && m.AmountVote.IsValid() && !m.AmountVote.IsZero() {
		return false
	}
	if !m.AmountDelegate.IsNil() && m.AmountDelegate.IsValid() && !m.AmountDelegate.IsZero() {
		return false
	}
	// all amount is empty
	return true
}
```

Reply:
done.